module/VuFind/src/VuFind/Config/Upgrade.php

```diff
@@ -133,20 +169,25 @@ public function run(string $newVersion): void
         // important since in some cases, settings may migrate out of config.ini
         // and into other files.
         $this->upgradeConfig($newVersion);
-        $this->upgradeAuthority();
         $this->upgradeFacetsAndCollection();
-        $this->upgradeReserves();
         $this->upgradeSearches();
         $this->upgradeSms();
         $this->upgradeEDS();
         $this->upgradeEPF();
         $this->upgradeSummon();
         $this->upgradePrimo();
-        $this->upgradeRecordDataFormatter();
 
         // The previous upgrade routines may have added values to permissions.ini,
         // so we should save it last. It doesn't have its own upgrade routine.
         $this->saveModifiedConfig('permissions', $this->permissionsModified);
+
+        foreach ($this->newConfigs as $filename => $newConfig) {
```

Comment:
Should we add a comment to explain this loop? (Alternatively, we could also move it to a function with a descriptive name; I'm fine with either approach).

```suggestion
        // Make sure to update any remaining configurations that were not explicitly updated above.
        foreach ($this->newConfigs as $filename => $newConfig) {
```